src/ui/Card/Card.tsx

```diff
@@ -0,0 +1,94 @@
+import { cva, VariantProps } from 'cva'
+import React from 'react'
+
+const card = cva(['border border-ds-gray-secondary'])
+interface CardProps
+  extends React.HTMLAttributes<HTMLDivElement>,
+    VariantProps<typeof card> {}
+
+const CardRoot = React.forwardRef<HTMLDivElement, CardProps>(
+  ({ className, ...props }, ref) => (
+    <div ref={ref} className={card({ className })} {...props} />
+  )
+)
+CardRoot.displayName = 'Card'
+
+const header = cva(['border-b', 'border-ds-gray-secondary', 'p-5'])
+interface HeaderProps
+  extends React.HTMLAttributes<HTMLDivElement>,
+    VariantProps<typeof header> {}
+
+const Header = React.forwardRef<HTMLDivElement, HeaderProps>(
+  ({ className, ...props }, ref) => (
+    <div ref={ref} className={header({ className })} {...props} />
+  )
+)
+Header.displayName = 'Header'
+
+const title = cva(['font-semibold'], {
+  variants: {
+    size: {
+      base: ['text-base'],
+      large: ['text-lg'],
+    },
+  },
+  defaultVariants: {
+    size: 'large',
+  },
+})
+interface TitleProps
+  extends React.HTMLAttributes<HTMLHeadingElement>,
+    VariantProps<typeof title> {}
+
+const Title = React.forwardRef<HTMLParagraphElement, TitleProps>(
+  ({ className, size, children, ...props }, ref) => (
+    <h3 ref={ref} className={title({ className, size })} {...props}>
+      {children}
+    </h3>
+  )
+)
+Title.displayName = 'title'
```

Comment:
Added a size variant bc my onboarding polish calls for a smaller title!

Reply:
Can you swap `large` for `lg`, makes it easier when we get into extra sizes `xl` etc.

Reply:
trueeee. Was wondering whether lg made more sense, but your argument has convinced me!